packages/experimental/Text/src/Variants.ts

```diff
@@ -0,0 +1,71 @@
+import { Variants } from '@fluentui-react-native/theme-types';
+import { Text } from './Text';
+import { createFontAliasTokens } from '@fluentui-react-native/win32-theme';
+
+const variantsWin32: Partial<Variants> = createFontAliasTokens('light');
```

Comment:
>('light'); [](http://example.com/codeflow?start=62&length=10)

win32 doesn't use light so I'm surprised this works. Should be one of Colorful/White/DarkGray/Black/HighContrast